File: mdc/DataCenter/MDC/Data/Dtos/UserSettingsDto.cs
namespace MDC.Data.Dtos
{
    public class UserSettingsDto : IdentifiedDto
    {
        public string Name { get; set; }

        public string Licenses { get; set; }

        public string Attributes { get; set; }

        public int Organisation { get; set; }

        public string World { get; set; }

        public double X { get; set; }

        public double Y { get; set; }

        public double Z { get; set; }
    }
}

File: server/plugins/MineParkCore/src/minepark/models/dtos/UserSettingsDto.php
<?php
namespace minepark\models\dtos;

class UserSettingsDto extends BaseDto
{
    public int $id;

    public string $name;

    public ?string $licenses;

    public ?string $attributes;
    
    public int $organisation;

    public ?string $world;

    public float $x;

    public float $y;

    public float $z;
}